loleaflet/src/control/Control.Menubar.js

```diff
@@ -1422,7 +1439,7 @@ L.Control.Menubar = L.Control.extend({
 			self._executeAction(item);
 		}
 
-		if (!window.mode.isMobile() && $(item).data('id') !== 'insertcomment')
+		if (!window.mode.isMobile() && $(item).data('id') !== 'insertcomment' && self && self.map)
 			self._map.focus();
```

Comment:
in condition: `self.map`
below: `self._map`

Reply:
Good catch! Pushed fix.